### PR TITLE
Add chrome.cookies.{getAll, set, get} for Pinterest Save Button extension, brave/browser-laptop#6366

### PR DESCRIPTION
**Pull request by kevinlawler**
See https://github.com/brave/vault-updater/pull/42

### CODE REVIEW

File: atom/browser/api/atom_api_importer.cc [hunk 1]
```diff
@@ -18,7 +18,7 @@
 #include "base/strings/utf_string_conversions.h"
 #include "base/threading/thread_task_runner_handle.h"
 #include "brave/browser/brave_content_browser_client.h"
-#include "chrome/browser/importer/external_process_importer_host.h"
+#include "brave/browser/importer/brave_external_process_importer_host.h"
```
Comment (bridiver) [inline atom/browser/api/atom_api_importer.cc]: I think these changes are unintentional? If you can clean them up I'll get it merged

**Comment by kevinlawler** [comment]
@bridiver rebased against master. lmk if you need any other changes

File: brave/browser/api/brave_api_component_updater.cc [hunk 2]
```diff
@@ -142,6 +142,9 @@ void ComponentUpdater::RegisterComponent(const std::string& component_id) {
   } else if (component_id == kHoneyId) {
     RegisterComponentForUpdate(
         kHoneyPublicKeyStr, registered_callback, ready_callback);
+  } else if (component_id == kPinterestId) {
```
Comment (bridiver) [inline brave/browser/api/brave_api_component_updater.cc]: @bbondy do we still need these? Can't we just register them in browser-laptop using the component api?

**Comment by kevinlawler** [comment]
split cookies fixes into separate branch/pr from pinterest key

**Pull request merged**